app/views/layouts/_institute_footer.html.erb

```diff
@@ -12,6 +12,7 @@
       </div>
       <div class="license">Licensed under the <a href="https://creativecommons.org/licenses/by-nc/2.0/" class="license-cc">Creative Commons Attribution Non-Commercial License</a> unless otherwise noted. <a href="https://libraries.mit.edu/research-support/notices/copyright-notify/">Notify us about copyright concerns</a>.
       </div><!-- end .footer-info-institute -->
+      <div>Access level: <%= access_level %></div>
```

Comment:
Yeah, this may get removed by UX. I won't be merging until I've confirmed with them. I warned @frrrances about it but haven't actually confirmed they do or do not want this in there.